src/test/java/neworangehrm/com/opensourcehrm/steps/AddUserSteps.java

```diff
@@ -0,0 +1,26 @@
+package neworangehrm.com.opensourcehrm.steps;
+
+import cucumber.api.PendingException;
+import cucumber.api.java.en.And;
+import neworangehrm.com.opensourcehrm.util.DriverUtil;
+import org.openqa.selenium.WebDriver;
+
+
+/**
+ * Created by Viswa on 12/27/2017.
+ */
+public class AddUserSteps {
```

Comment:
If this class is not used currently in the code then should be taken out as we are calling this 2.0 solid version

Reply:
Accepted and Fixed. Removed AddUserSteps class as we will baseline this code and call it as Version 1.0 and move on with further development of framework.